.github/workflows/release.yml

```diff
@@ -14,13 +14,15 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: Set up Python
-      uses: actions/setup-python@v5
+      uses: mamba-org/setup-micromamba@v1
       with:
-        python-version: 3.9
+        environment-file: environment-dev.yml
+        create-args: >-
```

Comment:
what does this >- syntax do?

Reply:
I don't know exactly, this is some YAML stuff, see https://github.com/mamba-org/setup-micromamba?tab=readme-ov-file#usage